src/components/pages/Offer/Offer/OfferDetails/__specs__/OfferEdition.spec.jsx

```diff
@@ -799,4 +804,82 @@ describe('offerDetails - Edition', () => {
       ).toBeInTheDocument()
     })
   })
+
+  describe('when clicking on cancel link', () => {
+    it('should call computeOffersUrl with proper params', async () => {
+      // Given
+      store = configureTestStore({
+        data: { users: [{ publicName: 'François', isAdmin: false }] },
+        offers: {
+          searchFilters: {
+            name: 'test',
+            offererId: 'AY',
+            venueId: 'EQ',
+            typeId: 'EventType.CINEMA',
+            status: 'all',
+            creationMode: 'manual',
+            periodBeginningDate: '2020-11-30T00:00:00+01:00',
+            periodEndingDate: '2021-01-07T23:59:59+01:00',
+            page: 1,
+          },
+        },
+      })
+      const editedOffer = {
+        id: 'ABC12',
+        name: 'My edited offer',
+        type: 'ThingType.LIVRE_EDITION',
+        description: 'Offer description',
+        venueId: venues[0].id,
+        withdrawalDetails: 'Offer withdrawal details',
+        bookingEmail: 'booking@example.net',
+        extraData: null,
+      }
+      pcapi.loadOffer.mockResolvedValue(editedOffer)
+      await renderOffers(props, store)
+
+      // When
+      userEvent.click(screen.getByText('Annuler'))
+
+      // Then
+      expect(computeUrl.computeOffersUrl).toHaveBeenLastCalledWith({
+        creationMode: 'manual',
+        name: 'test',
+        offererId: 'AY',
+        page: 1,
+        periodBeginningDate: '2020-11-30T00:00:00+01:00',
+        periodEndingDate: '2021-01-07T23:59:59+01:00',
+        status: 'all',
+        typeId: 'EventType.CINEMA',
+        venueId: 'EQ',
+      })
```

Comment:
```suggestion
      expect(computeUrl.computeOffersUrl).toHaveBeenLastCalledWith(store.offers.searchFilters)
```

Je crois que c'est l'objet tel qu'il provient du store qu'on passe à `computeOffersUrl` ? Dans ce cas, je verrais bien quelque chose de ce style qui simplifierait la lecture je pense. Mais je ne sais plus si on peut accéder si facilement à la donnée dans `store`

Reply:
Comme on construit le store dans nos tests via `configureTestStore`, on ne peut pas simplement faire un store.offers.searchFilters car il nous retourne undefined.
J'ai quand même extrait les valeurs fournies à la fonction `configureTestStore` dans une variable comme ça je peux utiliser la même dans mon expect pour mettre en exergue la corrélation entre les deux.